src/test/java/seedu/address/logic/commands/ClientNoteAddCommandTest.java

```diff
@@ -53,7 +61,34 @@ public void execute_notDuplicateClientNote_successfullyAddsClientNote() {
         } catch (CommandException e) {
             fail();
         }
+    }
+
+    @Test
+    public void equals() {
+        Index idx = Index.fromOneBased(1);
```

Comment:
Maybe `idx1` here since it's `clientNote1`

Reply:
Done